modules/ribbit/http.py

```diff
@@ -40,9 +41,19 @@ async def static(request):
             headers["Cache-Control"] = "public, max-age=604800, immutable"
 
         return data, 200, headers
-
-    @app.get("/api/config")
-    async def config_list(req):
+    
+    @app.route("/api/sensors")
+    @with_websocket
+    async def sensor_status(request, ws):
+        while True:
+            ret = collections.OrderedDict()
+            for sensor in registry.sensors.values():
+                ret[sensor.config.name] = sensor.export()
+
+            await ws.send(json.dumps(ret))
+
+    @app.route("/api/registry")
```

Comment:
This probably should stay `/api/config`.

Reply:
If this route is /api/config and the @app.patch('api/config') method remains below, will that conflict?

Reply:
They will not conflict: one is `GET`, the other is `PATCH`.